source/documentation/runbooks/remove-orphaned-form.html.md.erb

```diff
@@ -0,0 +1,43 @@
+---
+title: Removed an Ophaned Deployed Form
+category: runbooks
+last_reviewed_on: 2023-03-23
+review_in: 3 months
+---
+
+<% if current_page.url != "/" %># <%= current_page.data.title %><% end %>
+
+
+## Removed Ophaned Deployed Form
+
+SOmetime when making a change to a form, it isn't unpublihsed first. And could leave the pods running in the namespace.
```

Comment:
I think we might need to be a little more specific here, or at least I'm not clear on when we would run these commands.

We should outline the circumstances that could result in an orphan form. Broadly speaking, for this scenario, we could ask the user to re-pubilsh an existing form and that would delete/deploy new pods with the change.

Is this scenario just in the event someone asks for the form to be unpublished? I think we have an unpublish button in the Admin dashboard that can do this for you.

I'm not sure if this is just for Test or Live forms, but if the form is Live we will also need to remove Pingdom alerts before removing the pods.